src/providers/LocalVideoProvider/index.tsx

```diff
@@ -32,25 +33,50 @@ const LocalVideoProvider: React.FC = ({ children }) => {
       setIsVideoEnabled(true);
     }
 
+    const observer: AudioVideoObserver = {
+      videoAvailabilityDidChange: (availability) => {
+        if (!availability.canStartLocalVideo) {
+          setIsReachVideoLimit(true);
+        } else {
+          setIsReachVideoLimit(false);
+        }
+        console.log(
+          'video availability changed: canStartLocalVideo ',
+          availability.canStartLocalVideo
+        );
```

Comment:
Maybe we need to mention something about the # of video streams or video limit here.`video availability changed: canStartLocalVideo` doesn't seem to provide enough information here. Cause the variable you change here is about video limit.